platsec.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+ARTIFACTS_DIR="grype-vuln-artifacts"
+IMAGE_TAG="quay.io/gburges/kibana:latest"
```

Comment:
I would suggest to make this the first argument of the script with `$1` , and the artifacts dir the second, with a default vale ( something like `${2:-vuln-artifacts`}`

Then I'd delete and recreate the $ARTIFACTS_DIR every run.